src/detectors/exceptions/module.wat

```diff
@@ -4,10 +4,9 @@
 
 (module
   (func
-    (try
-      (catch
-        (drop)
-      )
-    )
+    try
```

Comment:
According to the [explainer](https://github.com/WebAssembly/exception-handling/blob/master/proposals/Exceptions.md), `try` needs a blocktype. Or is that optional?

Reply:
Syntax-wise that's optional, just like on regular blocks.

Reply:
Cool